signer/migrations/0003__create_tables.sql

```diff
@@ -152,6 +152,8 @@ CREATE TABLE sbtc_signer.completed_deposit_events (
     created_at          TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
+-- DEPRECATED: dropped in migration 0013__consolidate_withdrawal_tables.
+-- Use `sbtc_signer.withdrawal_requests` instead.
```

Comment:
We can keep this (and maybe even do something like this more often), but it's sometimes best not to modify old migration files. Some migration tools use a checksum to know if it should reapply a migration. We don't do that, so modifying a file is fine I just thought that it couldn't hurt for me to point out.